### PR TITLE
chore: update zeromq types to 5.2

### PR DESCRIPTION
**Pull request by overflowz**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/zeromq/zeromq.js/blob/5.x/lib/index.js
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by aminya — APPROVED** [review]
It generally looks good to me.

**Comment by overflowz** [comment]
@aminya also exposed the `Socket.closed` property (was missed previously).

**Review by aminya — APPROVED** [review]

**Comment by overflowz** [comment]
Any thoughts on why can't I merge? It's been 3 days that it says "🕐 Continuous integration tests are still running".

**Comment by aminya** [comment]
The maintainers should approve the pull request.
@davemckeown @erikma @TitaneBoy

**Comment by elibarzilay** [comment]
@overflowz: new GH policy requires approval for first-time contributors, and the bot needed to be adjusted for that.

File: types/zeromq/index.d.ts
Comment (aminya) [inline types/zeromq/index.d.ts:4]: @overflowz Do you want to add your name here?

Comment (overflowz) [reply types/zeromq/index.d.ts:4]: done 👌

**Review by aminya — APPROVED** [review]

**Comment by overflowz** [comment]
Ready to merge

**Pull request merged**
